### PR TITLE
Install norns in system location

### PR DESCRIPTION
**Pull request by simonvanderveldt**
So we can decouple it from the user's directory allowing for separate partitions in the future

### CODE REVIEW

**Comment by tehn** [comment]
this looks promising, i'll do some testing!

File: package/dust/sclang_conf.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+includePaths:
```
Comment (simonvanderveldt) [inline package/dust/sclang_conf.yaml:1]: Still need to fix https://github.com/monome/norns/issues/799 to get rid of this file.
It's the only thing left in `$HOME` apart from dust.

I think it should be possible using `recompile` from within sclang, but we'll need to figure out a way to add the path, recompile and only then run `Crone.initClass`.

**Comment by tehn** [comment]
i take it this also means we need a custom `norns/lua/core/config.lua` with correct paths?

https://github.com/monome/norns/blob/master/lua/core/config.lua#L7

**Comment by simonvanderveldt** [comment]
> i take it this also means we need a custom `norns/lua/core/config.lua` with correct paths?
> 
> https://github.com/monome/norns/blob/master/lua/core/config.lua#L7

@tehn I just hard patched it like this for now https://github.com/simonvanderveldt/norns-image/pull/12/files#diff-1825874b72bf4c0cd5fec63e89dda83eR22

**Comment by simonvanderveldt** [comment]
Working for me. In case someone runs into issues in the future we can fix/figure it out when it happens.

**Pull request merged**